packages/transaction-controller/src/TransactionController.test.ts

```diff
@@ -802,6 +967,27 @@ describe('TransactionController', () => {
       });
     });
 
+    it('fails to estimate gas', async () => {
```

Comment:
Why are we adding unrelated unit tests? Is it to stop the coverage decreasing?

If so, would it be simpler to let the coverage go down slightly in order to minimise the size and clarity of the PR, and then we can add additional unit tests in a separate dedicated PR if needed?

Reply:
Make sense, I will remove it and I will create a separate PR to increase the coverage.